frontend/e-buddy/src/components/Chat.tsx

```
@@ -93,4 +286,358 @@ async function handleSubmit(text?: string) {
+          message={message}
+          onFollowUpClick={onFollowUpClick}
+        />
+      );
+    case "comparison_table":
+      return (
+        <ComparisonTableDisplay
+          message={message}
+          onFollowUpClick={onFollowUpClick}
+        />
+      );
+    case "text":
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)